mamba/mamba/mamba_shell_init.py

```diff
@@ -96,13 +108,11 @@ def shell_init(args):
             args.reverse,
         )
         for el in plan:
-            if el.get("function") == "init_sh_user":
```

Comment:
I noticed the branches for `init_sh_user` and `init_sh_system` are the same. Moreover, a separate branch for `init_fish_user` would also be the same. So I tentatively merged them into one, under a single `if`. However, if separate branches are desirable for a reason I'm not aware of, I can keep the two existing ones and add a third separate one for `init_fish_user`.

Reply:
Looks good to me!